verification/verifier_test.go

```diff
@@ -0,0 +1,43 @@
+package verification
+
+import (
+	"strconv"
+	"testing"
+
+	corex509 "github.com/notaryproject/notation-core-go/x509"
+)
+
+func TestVerifyX509TrustedIdentities(t *testing.T) {
+
+	certs, _ := corex509.ReadCertificateFile("testdata/verifier/signing-cert.pem")
```

Comment:
It's not clear what the cert subject is that we are testing against, maybe add that as a comment and print it as part of test execution.